docs/sources/mimir/operators-guide/configure/configure-otel-collector.md

```diff
@@ -102,3 +102,25 @@ service:
       processors: [...]
       exporters: [..., otlphttp]
 ```
+
+## Format considerations
+
+We currently follow the official specification described here: [OTLP Metric points to Prometheus](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#otlp-metric-points-to-prometheus).
+
+The following sections describe common issues you might experience.
+
+#### Dots (.) are converted to _
+
+Because Prometheus metrics don't support `.` and `-` in metric or label names, they are converted to `_` in Prometheus.
+
+For example:
+
+`requests.duration{http.status_code=500, cloud.region=us-central1}` in OTLP becomes `requests_duration{http_status_code=”500”, cloud_region=”us-central1”}` in Prometheus.
```

Comment:
```suggestion
- Dots (`.`) are converted to `_`. Because Prometheus metrics do not support `.` and `-` characters in metric or label names, they get converted to `_` in Prometheus.

  For example:
  `requests.duration{http.status_code=500, cloud.region=us-central1}` in OTLP
  `requests_duration{http_status_code=”500”, cloud_region=”us-central1”}` in Prometheus
```

Reply:
The format needs to be test...